src/opnsense/mvc/app/views/OPNsense/Firewall/filter_rule.volt

```diff
@@ -706,9 +719,16 @@
         $('#category_filter').parent().find('.dropdown-toggle').prepend('<i class="fa fa-tag" style="margin-right: 6px;"></i>');
 
         $("#reconfigureAct").SimpleActionButton({
-            onAction: function (data, status) {
-                populateInterfaceSelectpicker();
-                populateCategoriesSelectpicker();
+            onAction(data, status) {
+                reconfigureActInProgress = true;
```

Comment:
technically, this should already be true on `onPreAction`, as `onAction` fires after the request is done.

`onPreAction` expects a promise to resolve though, so do `return Promise.resolve()` there.